docs/user/ppl/admin/cross_cluster_search.rst

```diff
@@ -0,0 +1,108 @@
+.. highlight:: sh
+
+====================
```

Comment:
suggest to add the doc in two place
1. search doc, https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/search.rst
2. identifier, https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/general/identifiers.rst#multiple-indices

Reply:
done

Reply:
should the cross_cluster_search be a section in search.rst?

Reply:
I see your point. make sense.